### ISSUE TITLE
Update guild join flow to be more useful

### ISSUE THREAD

**Issue by alalonde** [MetaFam/TheGame#1472]
### What would you like to be added?

- After connecting, button should just say "Join"
- More copy on the intro page explaining why going through Discord is necessary (from peth "better explain why its useful & what we'll be pulling in, currently only mentions announcements"). Currently we only describe why one might want to leave the permission box checked in the Discord flow
- Update final toast message to say something like "Thanks, your Guild will go live shortly!" 
- [Optional] On submission, post a message to champion's ring (or wherever appropriate) so someone can go into Hasura to activate the guild

### Why is this needed?

People will be manually sent a link to this page after they have already filled out a form and manually approved, so at this point they are pre-vetted and should be able to freely proceed.

However, anyone with the link could still input their data, so a manual approval process still seems useful

**Comment by cmaxwe11** [comment]
What worked:
1. Discord linking worked, Metagame bot is now a member of the server (what commands should I test with, what is expected to work on a non-Metagame Discord?)
2. Pulled in roles for me to select Admin/Member roles. 

I got stuck trying to submit guild info, a contract is required (which I believe should not be, as not all partner orgs will have on-chain DAOs). I tried just submitting a wallet address, and got this error:
![image](https://user-images.githubusercontent.com/116289711/205688459-df5f7077-1c42-498c-975f-bd3e31c1eade.png)
